spec/features/pet_applications/show_spec.rb

```diff
@@ -79,5 +88,15 @@
     expect(current_path).to eq("/pets/#{pet_1.id}")
     expect(page).to have_content("pending")
     expect(page).to have_content("On hold for #{app_1.name}")
+
+    visit "/pet_applications/#{app_1.id}"
+
+    within "section.application-#{app_1.id}-pets" do
+      find("#approve-#{pet_2.id}").click
+    end
+
+    expect(current_path).to eq("/pets/#{pet_2.id}")
+    expect(page).to have_content("pending")
+    expect(page).to have_content("On hold for #{app_1.name}")
   end
 end
```

Comment:
Awesome, we already had the functionality and just needed a test! 💯

Reply:
@SMJ289